x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/inventory_group_accordion.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useCallback, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+import { EuiAccordion, EuiPanel, EuiSpacer, useEuiTheme } from '@elastic/eui';
+import { InventoryGroupPanel } from './inventory_group_panel';
+import { GroupedGridWrapper } from './grouped_grid_wrapper';
+import { EntityGroup } from '../../../common/entities';
+import { InventoryPanelBadge } from './inventory_panel_badge';
+
+const ENTITIES_COUNT_BADGE = i18n.translate(
+  'xpack.inventory.inventoryGroupPanel.entitiesBadgeLabel',
+  { defaultMessage: 'Entities' }
+);
+
+export interface InventoryGroupAccordionProps {
+  group: EntityGroup;
+  groupBy: string;
+}
+
+export function InventoryGroupAccordion({ group, groupBy }: InventoryGroupAccordionProps) {
+  const { euiTheme } = useEuiTheme();
+  const field = group[groupBy];
+  const id = `inventory-group-${groupBy}-${field}`;
+  const [load, setLoad] = useState(false);
+  const [open, setOpen] = useState(false);
+
+  const onToggle = useCallback(() => {
+    setLoad(true);
+    setOpen((opened) => !opened);
+  }, []);
+
+  return (
+    <>
+      <EuiPanel hasBorder hasShadow={false} paddingSize="xs">
+        <EuiAccordion
+          className="inventoryGroupAccordion"
+          data-test-subj={id}
+          id={id}
+          buttonContent={<InventoryGroupPanel field={field} />}
+          buttonElement="div"
+          extraAction={
+            <InventoryPanelBadge
+              data-test-subj="inventory-panel-badge-entities-count"
+              name={ENTITIES_COUNT_BADGE}
+              value={group.count}
+            />
+          }
+          buttonProps={{ paddingSize: 'm' }}
+          paddingSize="none"
```

Comment:
Can't you just use paddingSize here?

```suggestion
          paddingSize="m"
```

Reply:
Doing that reduces the clickable area of the Accordion button. so it matches more what appears to be visually clickable (almost all of the accordion, not just the middle of it. I actually don't like how the button part of the accordion is small compared to the text part, feels like bad UX to have a mismatch of what feels like should be clickable but actually isn't.